ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1352,29 +1499,36 @@
           only_trigger_phrase: true
           trigger_permit_all: true
           status_context: jenkins-kind-ipv6-ds-conformance
-          status_url: --none--
+          status_url: null
           success_status: Build finished.
-          failure_status: Failed. Add comment /test-kind-aws-ipv6-conformance to re-trigger.
-          error_status: Failed. Add comment /test-kind-aws-ipv6-conformance to re-trigger.
+          failure_status: Failed. Add comment /test-kind-ipv6-conformance to re-trigger.
+          error_status: Failed. Add comment /test-kind-ipv6-conformance to re-trigger.
           triggered_status: null
           started_status: null
           wrappers:
           - timeout:
               fail: true
               timeout: 135
               type: absolute
-          publishers: []
+          publishers:
+          - archive:
+              allow-empty: true
+              artifacts: '*sonobuoy*.tar.gz'
+              case-sensitive: true
+              default-excludes: true
+              fingerprint: false
+              only-if-success: false
       - '{name}-{test_name}-for-pull-request':
-          test_name: kind-aws-ipv6-ds-e2e
-          node: 'antrea-kind-test'
+          test_name: kind-ipv6-ds-e2e
```

Comment:
test name could be mapped with same name like test-kind-ipv6-e2e to avoid confusion with trigger phrase !